pkg/pillar/cmd/zedrouter/pubsubhandlers.go

```diff
@@ -443,11 +443,15 @@ func (z *zedrouter) handleAppNetworkCreate(ctxArg interface{}, key string,
 		DisplayName:    config.DisplayName,
 	}
 	z.doCopyAppNetworkConfigToStatus(config, &status)
+	z.publishAppNetworkStatus(&status)
+	defer func() {
+		status.PendingAdd = false
```

Comment:
This defer is far away (several lines) from the set.
It is better to introduce a pattern and functions so one will have two lines next to eachother
   status.SetPendingAdd()
   defer status.ClearPendingAdd()
or something similar.

Reply:
Alternatively, do a point fix for this issue now and in LTS releases and explore a more regular pattern in the future.

Reply:
Done. I put `status.PendingAdd = true/false + z.publishAppNetworkStatus(&status)` next to each other.